### ISSUE TITLE
Some questions regarding the training

### ISSUE THREAD

**Issue by raywzy** [advimman/lama#111]
Amazing work!! The instructions about the released code are also really complete, thanks for sharing such a high-quality work!!!

I have one question about the training. Does BatchNormalization matter? In my experiments, I try to replace the ffc generator with the one in [AOT-GAN](https://github.com/researchmm/AOT-GAN-for-Inpainting/blob/418034627392289bdfc118d62bc49e6abd3bb185/src/model/aotgan.py#L44), which is a very simple CNN-based encoder-decoder architecture, and remove all the BN of discriminator. But the training seems totally fail and could not produce any results. Could you please give some insights about that?

**Comment by windj007** [comment]
Hi! Thank you for your interest in our work!

To my experience, BN is extremely important - we tried many different normalization techniques but could not match performance of the same architectures but with BN.

Did you try combining AOT-GAN generator (which is known to work) with Discriminator with BN (which is also known to work)?

**Comment by raywzy** [comment]
> Hi! Thank you for your interest in our work!
> 
> 
> 
> To my experience, BN is extremely important - we tried many different normalization techniques but could not match performance of the same architectures but with BN.
> 
> 
> 
> Did you try combining AOT-GAN generator (which is known to work) with Discriminator with BN (which is also known to work)?

Thanks for your great information!! Yes I will try it then😊

**Comment by cjt222** [comment]
@raywzy @windj007  Hi！Have you try AOT-GAN generator to get a great result? I try to replace ffc_resnet to AOT-GAN generator which reduce some channels. The model is trained on A100, the batch size is set to 30, and other configurations are set by default. I trained the model on place365 data and trained 100000 iters. The test found that the effect is still not good

**Comment by windj007** [comment]
Hi! I have not tried AOT-GAN generator in LaMa setup. 100k iterations seem to be too few for good quality. LaMa usually needs ~500k for near-convergence - but we train for 1M iters, because models keep improving.

**Comment by cjt222** [comment]
I see! Thanks for your reply! I will try again~

**Comment by Ellohiye** [comment]
> I see! Thanks for your reply! I will try again~

Hi, have you done the experiment as suggested by the author? What is the effect of adding the BN layer?Looking forward to your reply！

**Comment by Ellohiye** [comment]
> I see! Thanks for your reply! I will try again~

When I was training AOTGAN, this problem also occurred. The iterations increased, but the effect became bad. How about the effect of your training on AOTGAN? Very much looking forward to your reply!
